### ISSUE TITLE
Move Entry

### ISSUE THREAD

**Issue by kevinjfoley** [Kungsgeten/org-brain#121]
Is there any way to move an entry?

For example say I made the entry `programming` and then created a headline child entry of `python` and `emacs-lisp`.

I then decide I want to create a new `programming` child entry `languages` and make `python` and `emacs-lisp` children of `languages` so that they are no longer children of `programming` but instead become grand children.  Is that currently possible?

**Comment by Kungsgeten** [comment]
Hi!

That depends on the kind of entries you're trying to move. Headline entries (the ones with an ID) should probably be no worries. Moving file entries is trickier. In your case it sounds like `python` and `emacs-lisp` are both headline entries, and that `programming` is a file entry? It also sounds like they are "local children" to `programming` (meaning that they're simply headlines directly below `programming`; they're not added using the `BRAIN_CHILDREN` property).

If this is the case, you can just add the `languages` child to `programming`, and then open then org-file and simply move the headlines to be under `languages` instead. To be safe, make a backup of your org-brain directory first.

Long story short: There's currently no special command to move entries, but it isn't that hard to do so. It is somewhat trickier to move file entries. I haven't worked on `org-brain` for a couple of weeks now, but an upcoming feature is that file entries may have an ID, and in that case they'll be a bit easier to move.

**Comment by kevinjfoley** [comment]
Thanks for the info!  I ended up doing what you suggested and just shifted them in the org file.

That sounds great, in the meantime I'll see if I can come up with a function to accomplish this.

**Comment by Kungsgeten** [comment]
You can now move headline entries to other headline entries by using `org-brain-refile`.

**Comment by Kungsgeten** [comment]
The PR #221 will make it easier to move entries. In your example you would simply add `languages` as a parent to `python` and `emacs-lisp`, and then remove the `programming` parent from them.

**Comment by Kungsgeten** [comment]
The mentioned PR has been merged into the main branch. You can use `M p` to `M`ove the current entry to another `p`arent.
